discovery_engine_api_tests/tests/front_office/test_interactions.py

```diff
@@ -0,0 +1,48 @@
+
+import allure
+import pytest
+from handlers.api_handler import ApiHandler
+from model.interactions.interactions import Interactions
+from model.interactions.interaction import Interaction
+from utils import test_utils as tu
+from utils import assert_utils as au
+
+
+@allure.suite("Interactions Test Suite")
+class TestInteractions:
+    api_handler = ApiHandler()
+
+    @allure.severity(allure.severity_level.CRITICAL)
+    def test_positive_interaction(self, ingest_generated_document):
+        user_id = tu.generate_random_alphanumerical(6)
+        positive_interaction = Interactions(Interaction(id=ingest_generated_document, type="Positive")).to_json()
+        request = self.api_handler.interact_with_documents(user_id, positive_interaction)
+        au.assert_status_code_equals(request.status_code, 204)
+
+    @allure.severity(allure.severity_level.NORMAL)
+    @pytest.mark.skip(reason="todo: need to be updated")
```

Comment:
❓ does this refer to code in the engine itself, if so is there already a ticket for that? or does this refer to the id and interaction constructors in the test?

Reply:
@janpetschexain DMed you with explanation